lib/rubocop/ast/node/ensure_node.rb

```diff
@@ -6,18 +6,27 @@ module AST
     # node when the builder constructs the AST, making its methods available
     # to all `ensure` nodes within RuboCop.
     class EnsureNode < Node
-      # Returns the body of the `ensure` clause.
+      # Returns the body of the `ensure` node.
       #
-      # @return [Node, nil] The body of the `ensure`.
+      # @return [Node, nil] The body of the `ensure` node.
       def body
+        return rescue_node.body if rescue_node
+
+        node_parts[0]
+      end
+
+      # Returns an the ensure branch in the exception handling statement.
+      #
+      # @return [Node, nil] the body of the ensure branch.
+      def branch
```

Comment:
I used `branch` for the method to get the ensure branch body, similiar to how we have `branches` for `RescueNode`.